docker-compose.test.yml

```
@@ -3,15 +3,17 @@ services:
+      target: production
+    command: >
+      /bin/sh -c " echo '=== Container Start ===' && echo 'Current directory:' && pwd && echo 'Directory contents:' && ls -la && echo 'Out directory contents:' && ls -la out && echo '=== Starting serve ===' && npx serve -s out -p 3001 -v"
     restart: unless-stopped
     volumes:
       - .:/app
       - /app/node_modules
     ports:
       - '3001:3001'
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://${WEBSITE_DOMAIN}:${NEXT_PUBLIC_PROD_PORT}"]
+      test: [ "CMD", "curl", "-f", "http://${WEBSITE_DOMAIN}:${NEXT_PUBLIC_PROD_PORT}" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)